pkg/syncer/apis/edge/v1alpha1/edge-syncconfig.go

```diff
@@ -0,0 +1,78 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +crd
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:resource:scope=Cluster,shortName=esc
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type EdgeSyncConfig struct {
+	/*
+	 */
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   EdgeSyncConfigSpec   `json:"spec,omitempty"`
+	Status EdgeSyncConfigStatus `json:"status,omitempty"`
+}
+
+// EdgeSyncConfigSpec defines the desired state of EdgeSyncConfig
+type EdgeSyncConfigSpec struct {
+	// Downsncyed resource list
+	DownSyncedResources []EdgeSyncConfigResource `json:"downSyncedResources,omitempty"`
+
+	// Upsncyed resource list
+	UpSyncedResources []EdgeSyncConfigResource `json:"upSyncedResources,omitempty"`
+
+	// Conversions
+	Conversions []EdgeSynConversion `json:"conversions,omitempty"`
+}
+
+// Resource specifies down/up synced resource with exact GVK and name (and namespace if not cluster scoped resource)
+type EdgeSyncConfigResource struct {
+	// Kind of down/up synced resource
+	Kind string `json:"kind,omitempty"`
+
+	// Group of down/up synced resource
+	Group string `json:"group,omitempty"`
+
+	// Version of down/up synced resource
+	Version string `json:"version,omitempty"`
+
+	// Name of down/up synced resource
+	Name string `json:"name,omitempty"`
```

Comment:
When there is more than one object of a given kind in a given namespace, it would be more compact if a list of names could appear here.  If all objects of a given kind in a given namespace are desired, it would be more compact if `"*"` could appear here.